doc/getting_started/getting_started.rst

```diff
@@ -77,8 +77,13 @@ Clone the Zephyr source code repositories from GitHub using the ``west`` tool:
    If you had previously cloned the zephyr repository manually using Git,
    move the cloned :file:`zephyr/` repository to an empty enclosing folder
    (for example :file:`zephyrproject/zephyr/`), and
-   from this empty enclosing folder :file:`zephyrproject/`
-   run ``west init`` and then ``west clone``.
+   from this empty enclosing folder :file:`zephyrproject/` run::
```

Comment:
Just nitpicking here.
If you just moved `zephyr` into the enclosing folder `zephyrproject`, then I assume it's no longer empty ?

Reply:
Good catch. Will fix